packages/paste-core/components/time-picker/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {formatReturnTime} from '../src/utils';
+import {
+  DefaultTimePicker,
+  DisabledTimePicker,
+  ErrorTimePicker,
+  InverseTimePicker,
+  ReadonlyTimePicker,
+} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+describe('formatReturnTime()', () => {
+  const TEST_TIME = '13:30:04';
+
+  it('should format times', () => {
+    expect(formatReturnTime(TEST_TIME, 'HH:mm')).toEqual('13:30');
+    expect(formatReturnTime(TEST_TIME, 'hh:mm')).toEqual('01:30');
+    expect(formatReturnTime(TEST_TIME, 'HH:mm:ss')).toEqual('13:30:04');
+    expect(formatReturnTime(TEST_TIME, 'hh:mm:ss')).toEqual('01:30:04');
+    expect(formatReturnTime(TEST_TIME, 'HH:mm:ss.SSS')).toEqual('13:30:04.000');
+    expect(formatReturnTime(TEST_TIME, 'HH:mm:ss.SSS')).toEqual('13:30:04.000');
+    expect(formatReturnTime(TEST_TIME, 'hh:mm:ss.SSS aa')).toEqual('01:30:04.000 p.m.');
+    // testing midnight
+    expect(formatReturnTime('00:00', 'hh:mm A')).toEqual('12:00 AM');
+    // test for edge cases (passing in time+date value) to throw an error
+    expect(() => {
```

Comment:
Have you considered breaking down this `it` statement? In my experience, it's easier to debug test failures when there are less `expect` statements within an `it` block. For example, we could move the edge cases and the expected thrown error statement to a new block.

Reply:
Thanks for noticing this! I pulled out the last 2 expects in `time-picker` and `date-picker`.